cpp/open3d/visualization/gui/BitmapWindowSystem.h

```diff
@@ -27,6 +27,7 @@ struct TextInputEvent;
 
 class BitmapWindowSystem : public WindowSystem {
 public:
+    // Not needed anymore - may be removed
```

Comment:
Comment this out, if we do not need it any more, but want to keep it as reference.